packages/ezflow_extension/ezextension/ezflow/classes/ezflowoperations.php

```diff
@@ -73,6 +73,7 @@ public static function updateBlockPoolByBlockID( $block, $publishedBeforeOrAt =
         $fetchClassOptions->iniFile = 'block.ini';
         $fetchClassOptions->iniSection = $block['block_type'];
         $fetchClassOptions->iniVariable = 'FetchClass';
+        $fetchClassOptions->handlerParams = compact('block');
```

Comment:
I would rather pass an array or some kind of struct, as won't be able to introduce more params without BC in future.